devel/tests/test_reporead.py

```diff
@@ -39,3 +42,26 @@ def test_read_packages(self):
         packages = Package.objects.all()
         import_packages = ["{}-{}-{}".format(pkg.pkgname, pkg.pkgver, pkg.pkgrel) for pkg in packages]
         self.assertItemsEqual(files, import_packages)
+
+    def test_flagoutofdate(self):
+        # Add Pkg
+        # add Out of date
+        repo = Repo.objects.get(name__iexact='core')
+        arch = Arch.objects.get(name__iexact='any')
+
+        now = datetime.now(tz=timezone.utc)
+        pkg = Package.objects.create(arch=arch, repo=repo, pkgname='systemd',
+                                     pkgbase='systemd', pkgver='4.10.1',
+                                     pkgrel='1', pkgdesc='Linux kernel',
+                                     compressed_size=10, installed_size=20,
+                                     last_update=now, created=now)
+
+        FlagRequest.objects.create(pkgbase=pkg.pkgbase, repo=pkg.repo,
+                                   pkgver=pkg.pkgver, epoch=pkg.epoch,
+                                   ip_address='1.1.1.1')
+
+        with patch('devel.management.commands.reporead.logger') as logger:
+            call_command('reporead', 'x86_64', 'devel/fixtures/core.db.tar.gz')
+        logger.info.assert_called()
+        
+        self.assertEquals(len(FlagRequest.objects.all()), 0)
```

Comment:
This should include a test for a community-testing or a different repo and check that a newer version in repo A doesn't remove a flag request for repo B.